libpod/networking_linux.go

```diff
@@ -262,6 +297,21 @@ func (r *Runtime) setupRootlessNetNS(ctr *Container) error {
 		cmdArgs = append(cmdArgs, "--enable-seccomp")
 	}
 
+	if outboundAddr != "" {
+		if !slirpFeatures.HasOutboundAddr {
+			return errors.Errorf("outbound_addr not supported")
+		}
+		cmdArgs = append(cmdArgs, fmt.Sprintf("--outbound-addr=%s", outboundAddr))
+	}
+
+	if outboundAddr6 != "" {
+		if !slirpFeatures.HasOutboundAddr {
+			return errors.Errorf("outbound_addr6 not supported")
+		}
+		cmdArgs = append(cmdArgs, fmt.Sprintf("--outbound-addr6=%s", outboundAddr6))
+		cmdArgs = append(cmdArgs, "--enable-ipv6")
```

Comment:
We should also have a way to enable ipv6 without explicitly specifying the outbound address

Reply:
Alright I exposed `enable_ipv6`. I was not sure about it being automatically enabled with `outbound_addr6`. It does not feel transparent enough imo (I am not fan of enabling flags because of others require them). So right now it works only when `enable_ipv6=true` otherwise shows error with explanation. 
Is it sufficient or would you like me to implement it some other way? @AkihiroSuda

Reply:
Looks good